### DISCUSSION TITLE
Feature request regarding export window

### DISCUSSION THREAD

**Discussion by perfrank** [UtilitechAS/amsreader-firmware#793] [Ideas]
Export window is only visibel when actually exporting.
It would be nice to be able to have it permanently visible, as I use it a lot for a quick overview.
Could be controlled by a tick-off in settings.
Else a great product!

**Comment by ArnieO** [comment]
Hi, 
You can achieve exactly that in the "User interface" tile on the config page. 
Simply change settings from "Auto" to "Enabled". 😉 
![image](https://github.com/UtilitechAS/amsreader-firmware/assets/10295178/804d41eb-0b49-435a-9d26-4052bbbd1d51)

**Comment by perfrank** [comment]
Sorry, I was imprecise in my request. What I meant was  the Real time
calculation window.. When not exporting, is only shows "Consumption" and
"Cost", and I would like it to show "Import" and "Export" permanently.

Sorry for.the confusion.


Med venlig hilsen / best regards

Per Frank-Hansen
Søvej 3
2665 Vallensbæk Strand
+45 20837692


Den tirs. 14. maj 2024 kl. 11.38 skrev ArnieO ***@***.***>:

**Reply by Noschvie** [reply]
After sunset / exporting the "Export" fields of "Real time calculation" disappear.

**Reply by ArnieO** [reply]
Ok, I see - and that is indeed a well-founded request and a good enhancement idea. 
I think an improved logic here could be:

If the "Production" parameter in the "Meter" tile in the config is non-zero:
- The "Export gauge" should be set to "Enabled" (not "Auto")
- The "Export" section of the "Real time calculation" tile should always be shown

Agreed?

I'll make sure this enhancement gets noted for the next firmware revision. 
Thank you for your help!

**Comment by perfrank** [comment]
Exactly. And I want it to stay;-)

Med venlig hilsen / best regards

Per Frank-Hansen
Søvej 3
2665 Vallensbæk Strand
+45 20837692


Den tirs. 14. maj 2024 kl. 12.19 skrev Norbert ***@***.***>:

**Comment by perfrank** [comment]
Yes, agreed! 😀
Thank You.

Med venlig hilsen / best regards

Per Frank-Hansen
Søvej 3
2665 Vallensbæk Strand
+45 20837692


Den tirs. 14. maj 2024 kl. 12.36 skrev ArnieO ***@***.***>:

**Comment by gskjold** [comment]
Judging from this discussion and my current testing, you have not configured "Production" under "Meter" on the config page. Configure your peak production capacity here, and all the export data will be visible even after sunset

**Comment by perfrank** [comment]
Yes, that did the trick. Thank You very much.

Med venlig hilsen / best regards

Per Frank-Hansen
Søvej 3
2665 Vallensbæk Strand
+45 20837692


Den tors. 23. maj 2024 kl. 16.50 skrev Gunnar Skjold <
***@***.***>:
